src/composables/transactionTokens.ts

```diff
@@ -63,24 +63,26 @@ export function useTransactionTokens({
       }));
     }
 
+    const isReceived = direction === TX_DIRECTION.received;
+    const protocolAdapter = ProtocolAdapterFactory
```

Comment:
I suggest to rename this to `adapter` as we use only one type of adapters within the system and the `adapter` constant is already more common.